ui/shared/Revision.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-regular-svg-icons';
+import { Row } from 'reactstrap';
+
+import { parseAuthor } from '../helpers/revision';
+
+import BugLinkify from './BugLinkify';
+import Clipboard from './Clipboard';
+
+export function Initials(props) {
```

Comment:
I think `AuthorInitials` would be a little clearer.

Reply:
Good call, yeah.  I'll fix it.